source/maze/MouseController.cpp

```diff
@@ -193,7 +193,7 @@ bool MouseController::isInStart()
 void MouseController::printMaze()
 {
     Maze::updateCell();
-    mazeIO.setMousePosition(getCurrentPos());
+    mazeIO.setMousePosition(PositionController(getCurrentPos(), getCurrentDir()));
```

Comment:
You don't need to pass a new `PositionController`. Since `MouseController` inherits `PositionController`, you can pass its object itself.